tools/pkg-helpers/pytorch_pkg_helpers/cuda.py

```diff
@@ -48,6 +48,7 @@ def get_cuda_variables(
         ret.extend(
             [
                 f"export CUDA_HOME='{cuda_home}'",
+                f"export CUDA_PATH='{cuda_home}'",
```

Comment:
curious what required this change

Reply:
This was an issue building cuda 11.7 wheels here: https://github.com/pytorch/test-infra/actions/runs/4502882132
Since we have both cuda 11.7 and 11.8 installed on this AMI, somehow the cuda 11.8 is getting picked up if this variable not set.

It actually mentions it here: https://github.com/pytorch/pytorch/blob/416bac5b813a181753afade781ae30f4f0843586/setup.py#L153